File: app/src/main/java/com/justaclock/viewmodels/MainViewModel.java
package com.justaclock.viewmodels;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;
import androidx.lifecycle.MutableLiveData;

import com.justaclock.adapters.Task;

import java.util.ArrayList;

public class MainViewModel extends AndroidViewModel {
    public MutableLiveData<String> currentFragment;

    public MainViewModel(@NonNull Application application) {
        super(application);
        currentFragment = new MutableLiveData<>();
    }
}

File: app/src/main/java/com/justaclock/viewmodels/MainViewModel.kt
package com.justaclock.viewmodels

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import androidx.lifecycle.MutableLiveData

class MainViewModel(application: Application) : AndroidViewModel(application) {
    var currentFragment: String? = null
    var isTheFirstTimeInTheApp: Boolean? = true
    var lastFragment: String? = null
}
